components/openindiana/bhyve-fw/Makefile

```diff
@@ -30,17 +30,19 @@ COMPONENT_ARCHIVE_HASH= \
 COMPONENT_ARCHIVE_URL= \
 	https://github.com/illumos/$(COMPONENT_NAME)/releases/download/$(COMPONENT_TAG)/$(COMPONENT_ARCHIVE)
 
-COMPONENT_TAG_1=	il-edk2-stable202102-1
+COMPONENT_TAG_1=	il-edk2-stable202308-1
 COMPONENT_SRC_1=	$(COMPONENT_NAME)-$(COMPONENT_TAG_1)
 COMPONENT_ARCHIVE_1=	$(COMPONENT_NAME)-$(COMPONENT_TAG_1).tar.xz
 COMPONENT_ARCHIVE_HASH_1= \
-	sha256:ca26699ec8f8f7e45e01f2d5a23649997d33f0b8437405ca731f6e301d6a71a8
+	sha256:8aaacf4b7c6a875c78492ad74856802abcbce89ee8cf980efbd12c74946b3b79
 COMPONENT_ARCHIVE_URL_1= \
 	https://github.com/illumos/$(COMPONENT_NAME)/releases/download/$(COMPONENT_TAG_1)/$(COMPONENT_ARCHIVE_1)
 
 COMPONENT_PROJECT_URL=	https://github.com/illumos/uefi-edk2
 COMPONENT_LICENSE=	simplified-BSD
 
+GCC_VERSION= 13
```

Comment:
This is not enough to switch from illumos-gcc to gcc-13.  Below you force `GCCPATH=/opt/gcc/4.4.4` so you effectively do not use gcc-13 I assume.  Or, is it used?

Reply:
Ah I see.  `GCCPATH` is set to `/opt/gcc/4.4.4` for csm builds only.  For non-csm you use gcc-13.

Then, this line is not needed, because it is default.  Or, is it critical to use exactly gcc-13 for non-csm builds and nothing newer or older?  If basically any gcc could be used, then this line should not be there.

Thank you.